### ISSUE TITLE
Menu is hidden on Safari

### ISSUE THREAD

**Issue by adneforslund** [equinor/energyvision#515]

**Comment by nilsml** [comment]
Seems like we have a few issues with Safari, the most important browser 😜

**Comment by wenche** [comment]
I'm working on it as we speak, so this is just a temp. state :)

**Comment by adneforslund** [comment]
![StatCounter-browser-NO-monthly-202008-202108](https://user-images.githubusercontent.com/25346938/135255049-0ab8b7ae-bd36-4f7b-8ce3-752a8fca5c01.png)

**Comment by SvSven** [comment]
https://www.safari-is-the-new-ie.com

**Comment by nilsml** [comment]
So this one is fixed now? @adneforslund

**Comment by adneforslund** [comment]
@nilsml yes, and also I think this issue is redundant, and probably should not have been a issue in the first place
